### PR TITLE
Last online attribute

### PR DESCRIPTION
**Pull request by darren2pro**
Ok I think should be good for review

Shall I make a resetLastOnline to set it to nil if we have already synced?

### CODE REVIEW

File: Annotato/Annotato/WebSocket/WebSocketManager.swift [hunk 1]
```diff
@@ -3,6 +3,7 @@ import AnnotatoSharedLibrary
 
 class WebSocketManager {
```
Comment (sivayogasubramanian) [inline Annotato/Annotato/WebSocket/WebSocketManager.swift]: Can we put the last online time related things in a extension similar to AnnotatoAuth

**Review by sivayogasubramanian — COMMENTED** [review]
LGTM! Some minor refactoring and it will be good to merge

File: Annotato/Annotato/WebSocket/WebSocketManager.swift (hunk 1 above)
Comment (darren2pro) [reply Annotato/Annotato/WebSocket/WebSocketManager.swift]: Ok done

**Review by sivayogasubramanian — APPROVED** [review]
LGTM! Thanks

**Pull request merged**